### ISSUE TITLE
Sample Code for Webhooks

### ISSUE THREAD

**Issue by fundthmcalculus** [Dentity-Development/dentity-ecosystems-sdk#747]
Sample code for webhooks

Top Priority
- [x] .NET
- [x] Typescript
- [x] CLI
  - Blocked until #769 is handled

Next priority
- [x] Go
- [x] Java
- [x] Python
- [x] Ruby
- [x] Swift
- [x] Dart

**Comment by geel9** [comment]
CLI doesn't currently support `AddWebhook`, `DeleteWebhook`, or `AuthorizeWebhook`

**Comment by geel9** [comment]
Thoughts:

- CLI must implement support for these endpoints before we can write sample code
- Swift and Dart samples are not currently included in documentation.
  - Therefore, these are not blocking languages, until/unless we elevate them to first-class languages
- Ruby samples are included in _some_ places, but not others
  - We need to decide if Ruby is a first-class language.
    - If so, we need to create a new ticket to create/inject samples *everywhere* for Ruby, instead of a random assortment of places
    - If not, we need to remove Ruby samples from wherever they currently are.

**Comment by fundthmcalculus** [comment]
* Swift is officially supported, just not doc-injected because of swift package manager requiring a [separate repository](https://github.com/trinsic-id/sdk-swift).
* Ruby samples should probably just be hidden for the time being, we can add them back in post GA. Some people are interested in Ruby, but not in the 1-month timeframe.
* Dart samples don't need to exist, at least not yet.

**Comment by geel9** [comment]
Closed by #785
